src/java/org/jivesoftware/openfire/http/HttpSessionManager.java

```diff
@@ -81,39 +81,13 @@ public HttpSessionManager() {
     	
         JiveGlobals.migrateProperty("xmpp.httpbind.worker.threads");
         JiveGlobals.migrateProperty("xmpp.httpbind.worker.timeout");
-    	
-        this.sessionManager = SessionManager.getInstance();
-        init();
     }
 
-    public void init() {
-        Log.warn("HttpSessionManager.init() recreate sendPacketPool");
-        // Configure a pooled executor to handle async routing for incoming packets
-        // with a default size of 16 threads ("xmpp.httpbind.worker.threads"); also
-        // uses an unbounded task queue and configurable keep-alive (default: 60 secs)
-        
-        // Note: server supports up to 254 client threads by default (@see HttpBindManager)
-        // BOSH installations expecting heavy loads may want to allocate additional threads 
-        // to this worker pool to ensure timely delivery of inbound packets
-        
-        int maxPoolSize = JiveGlobals.getIntProperty("xmpp.httpbind.worker.threads", 
-				// use deprecated property as default (shared with ConnectionManagerImpl)
-				JiveGlobals.getIntProperty("xmpp.client.processing.threads", 8));
-        int keepAlive = JiveGlobals.getIntProperty("xmpp.httpbind.worker.timeout", 60);
-
-        sendPacketPool = new ThreadPoolExecutor(getCorePoolSize(maxPoolSize), maxPoolSize, keepAlive, TimeUnit.SECONDS, 
-			new LinkedBlockingQueue<Runnable>(), // unbounded task queue
-	        new ThreadFactory() { // custom thread factory for BOSH workers
-	            final AtomicInteger counter = new AtomicInteger(1);
-	            @Override
-	            public Thread newThread(Runnable runnable) {
-	                Thread thread = new Thread(Thread.currentThread().getThreadGroup(), runnable,
-	                                    "httpbind-worker-" + counter.getAndIncrement());
-	                thread.setDaemon(true);
-	                return thread;
-	            }
-	    	});
-    }
+    /**
+     * @deprecated As of Openfire 4.0.0, the functionality of this method was added to the implementation of #start().
+     */
+    @Deprecated
+    public void init() {}

```

Comment:
I did not want to remove init() because it is declared public. That's just me being cautious, perhaps overly so. Doesn't hurt to keep it around for a release or so.